save-cloud-common/src/commonMain/kotlin/org/cqfn/save/entities/Annotations.kt

```diff
@@ -22,7 +22,18 @@ expect annotation class GeneratedValue()
 
 @OptIn(ExperimentalMultiplatform::class)
 @OptionalExpectation
-expect annotation class JoinColumn()
+@Suppress("KDOC_NO_CONSTRUCTOR_PROPERTY", "LongParameterList")
+expect annotation class JoinColumn(
```

Comment:
some ideas under this changes?

Reply:
I wanted to use `JoinColumn` in common code, but it turned out to be impossible because of `foreignKey` parameter. IIRC, we had some troubles with it before too, so I decided to keep my changes here so that we don't re-implement them some more time later. Generally, we should move `Project` into `jvmMain` and remove these `OptionalExpectation`s completely.